src/reducers.js

```diff
@@ -14,8 +14,9 @@ const createRootReducer = history =>
     // The authentication state is added as initialState when
     // creating the store in data/store.js.
     authentication: identityReducer,
+    configuration: identityReducer,
     userAccount,
-    profilePage,
+    profilePage: profileReducer,
```

Comment:
In a world where profilePage was a plugin and/or we wanted to have a common MFE boilerplate module that was shared across MFEs, you could imagine we do something fancy to remove this direct dependency on the profile reducer.  I'm not sure what that looks like, but it's basically part of a plugin architecture for MFEs.